SCClassLibrary/Common/Control/Server.sc

```diff
@@ -1,119 +1,63 @@
 ServerOptions {
-	classvar <defaultValues;
+	// DO NOT CHANGE THE OREDER OF THESE INSTANCE VARIABLES
+	// They are used by _BootInProcessServer
+	var <numAudioBusChannels = 1024;
+	var <>numControlBusChannels = 16384;
+	var <numInputBusChannels = 2;
+	var <numOutputBusChannels = 2;
+	var <>numBuffers = 1024;
 
-	// order of variables is important here. Only add new instance variables to the end.
-	var <numAudioBusChannels;
-	var <>numControlBusChannels;
-	var <numInputBusChannels;
-	var <numOutputBusChannels;
-	var <>numBuffers;
+	var <>maxNodes = 8192;
+	var <>maxSynthDefs = 8192;
+	var <>protocol = \udp;
+	var <>blockSize = 64;
+	var <>hardwareBufferSize = 0;
```

Comment:
I think this is incorrect: it effectively requests the _smallest_ possible buffer size on macOS, instead of the default buffer size. I think it should be `nil` by default, so that the server chooses the default buffer size. 

_Alternatively_, we'd need to change the server behavior so that it interprets `0` as request for the default buffer size, but that may be a breaking change?

On my system (macOS) this results in a buffer size of 15 samples (???) and the test suite indeed breaks with that setting (why it breaks and why it defaults to such a buffer size may be an issue to address in another PR).